src/modules/initcpiocfg/main.py

```diff
@@ -21,20 +21,19 @@
 #   You should have received a copy of the GNU General Public License
 #   along with Calamares. If not, see <http://www.gnu.org/licenses/>.
 
-import libcalamares
 from libcalamares.utils import debug, target_env_call
 import os
 from collections import OrderedDict
 
 import gettext
-_ = gettext.translation("calamares-python",
+translate = gettext.translation("calamares-python",
                         localedir=libcalamares.utils.gettext_path(),
                         languages=libcalamares.utils.gettext_languages(),
                         fallback=True).gettext
 
 
 def pretty_name():
-    return _("Configuring mkinitcpio.")
+    return translate("Configuring mkinitcpio.")
 
 
 def cpuinfo():
```

Comment:
This whole function is called once, and then the information it digs up is used once, in exactly one way (to find the vendor_id of proc0). I think replacing it with a `is_intel_cpu()` that returns bool (and then patching up the one place the information is used) is worthwhile.